classes/tax/TaxRulesGroup.php

```diff
@@ -134,22 +134,64 @@ public function getIdTaxRuleGroupFromHistorizedId($id_tax_rule)
 
     public static function getTaxRulesGroups($only_active = true)
     {
-        return Db::getInstance()->executeS('
-			SELECT DISTINCT g.id_tax_rules_group, g.name, g.active
-			FROM `' . _DB_PREFIX_ . 'tax_rules_group` g'
-            . Shop::addSqlAssociation('tax_rules_group', 'g') . ' WHERE deleted = 0'
-            . ($only_active ? ' AND g.`active` = 1' : '') . '
-			ORDER BY name ASC');
+        return static::getTaxRulesGroupsData($only_active, false);
     }
 
     /**
-     * @return array an array of tax rules group formatted as $id => $name
+     * @return array an array of tax rules group formatted as:
+     *               [
+     *               [
+     *               'id_tax_rules_group' => ...,
+     *               'name' => ...,
+     *               'active' => ...,
+     *               'rate' => ...,
+     *               ],
+     *               ...
+     *               ]
```

Comment:
This looks weird, was it cs fixer?

Reply:
Yes I'll try to reformat it better in respect with cs fixer